README.md

```diff
@@ -1,72 +1,195 @@
+	}
+    ```
 
-            _context.Set(configuration);
+### 3. Set up your object context
 
-            var executionConfig = new EnvironmentConfig { EnvironmentName = _configurator.EnvironmentName, ProjectName = _configurator.ProjectName };
+    ```
+    private ScenarioContext Context { get; set; }
+    
+    public void SetUpObjectContext()
+    {
+	    var objectContext= new ObjectContext();
+		this.Context.SetObjectContext(objectContext);
+	}
+    ```
+	
+### 4. Set up your Selenuim Webdriver
+
+    ```
+    private ScenarioContext Context { get; set; }
+    
+    public void SetUpWebDriver()
+    {
+	    var webdriverSupport = new WebDriverSupport<AppSettings>(this.Context);
+	    var webDriver = webdriverSupport.Create();
+	    this.Context.SetWebDriver(webDriver);
+    }
+    ```
 
-            _context.Set(executionConfig);
+### 5. Set up the helper library
 
-            var testExecutionConfig = _configSection.GetConfigSection<TestExecutionConfig>();
+    ```
+    private ScenarioContext Context { get; set; }
+    
+    public void SetUpHelperLibrary()
+    {
+	    var webDriver = this.Context.GetWebDriver();
+	    var settingsLibrary = this.Context.GetSettingsLibrary<AppSettings>();
+	    var helperLibrary = new HelperLibrary<AppSettings>(webDriver, settingsLibrary);
+	    this.Context.SetHelperLibrary(helperLibrary);
+    }
+    ```
 
-            _objectContext.SetBrowser(testExecutionConfig.Browser);
+### Other useful examples
 
-            var config = _configSection.GetConfigSection<ContactUs>();
-            _context.SetContactUsConfig(config);
+    ```
+    private ScenarioContext Context { get; set; }
+    
+    public void TakeAScreenshot() 
+    {
+	    var helperLibrary = this.Context.GetHelperLibrary<AppSettings>();
+		helperLibrary.ScreenshotHelper.TakeScreenshot(this.Context);
+    }
 
-            var mongoDbconfig = _configSection.GetConfigSection<MongoDbConfig>();
-            _context.SetMongoDbConfig(mongoDbconfig);
+    public void SelectARadioButton() 
+    {
+	    var radioButtonLocator = By.Id("radio_button_id");
+        var formHelper = this.context.GetHelperLibrary<AppSettings>().FormHelper;
+        formHelper.SelectRadioButton(radioButtonLocator);   
+    }
+    
+    public void GetTextFromAnIWebElement() 
+    {
+	    var webElementLocator = By.CssSelector("my_css_selector");
+	    var helperLibrary = this.Context.GetHelperLibrary<AppSettings>();
+		var elementText = helperLibrary.CommonActionHelper.GetText(webElementLocator);
+    }
 
-            _objectContext.Replace("browser", config.Browser);
-            _objectContext.Replace("build", config.BuildNumber);
-            _objectContext.Replace("EnvironmentName", config.EnvironmentName);
-            
-            
-        }
+    public void RunAxeAccessibilityAnalysis() 
+    {
+	    var axeHelper = this.Context.GetHelperLibrary<AppSettings>().AxeHelper;
+        axeHelper.Analyse();    
     }
-```
+    ```
+
+# Support
```

Comment:
@jbarnes16503 Can you strip this section out please.

Reply:
Done. Thanks.